src/fuzzer/tls_13_handshake_layer.cpp

```diff
@@ -0,0 +1,40 @@
+/*
+* (C) 2022 Jack Lloyd
+* (C) 2002 René Meusel - neXenio GmbH
+*
+* Botan is released under the Simplified BSD License (see license.txt)
+*/
+
+#include "fuzzers.h"
+#include <botan/internal/tls_handshake_layer_13.h>
+#include <botan/internal/tls_transcript_hash_13.h>
+
+
+namespace {
+
+Botan::TLS::Handshake_Layer prepare(const std::vector<uint8_t>& data)
+   {
+   Botan::TLS::Handshake_Layer hl(Botan::TLS::Connection_Side::CLIENT);
+   hl.copy_data(data);
+   return hl;
+   }
+
+}  // namespace;
+
+
+void fuzz(const uint8_t in[], size_t len)
+   {
+   static Botan::TLS::Default_Policy policy;
+
+   try
+      {
+      std::vector<uint8_t> v(in, in + len);
```

Comment:
@reneme I'm unable to push the following change (403) to the vector's type:
```suggestion
Botan::TLS::Handshake_Layer prepare(const Botan::secure_vector<uint8_t>& data)
   {
   Botan::TLS::Handshake_Layer hl(Botan::TLS::Connection_Side::CLIENT);
   hl.copy_data(data);
   return hl;
   }

}  // namespace;


void fuzz(const uint8_t in[], size_t len)
   {
   static Botan::TLS::Default_Policy policy;

   try
      {
      Botan::secure_vector<uint8_t> v(in, in + len);
```

Can you approve the suggestion so the build can hopefully pass?

Reply:
I decided to open a new PR anyway, the fuzzer now works as intended: #3013 

I suggest closing this PR (I apparently can't do this myself) in favor of the other one, because (a) the other one is ready to merge (b) I can rebase the other PR if something else changes in the meantime.